.jscsrc

```diff
@@ -1,5 +1,6 @@
 {
 	"preset": "jquery",
 	"maximumLineLength": null,
-	"requireCamelCaseOrUpperCaseIdentifiers": null
+	"requireCamelCaseOrUpperCaseIdentifiers": null,
+	"requireCapitalizedComments": false
```

Comment:
I forgot to mention, I added this property only because of this lines: [/test/methods.js#L993-L994](https://github.com/jzaefferer/jquery-validation/blob/master/test/methods.js#L993-L994).

JSCS complain about that this comment start with a lower case letter. so I disabled the requirement for comments to start with upper case (but it's against the rules: http://contribute.jquery.org/style-guide/js/#comments).

So, what do you think ?